src/clj/puppetlabs/services/request_handler/request_handler_service.clj

```diff
@@ -26,13 +26,9 @@
                               config)
                             current-code-id)]
       (when (contains? (:master config) :allow-header-cert-info)
-        (if (true? (get-in config [:jruby-puppet :use-legacy-auth-conf]))
-          (log/warn (format "%s %s"
-                            (i18n/trs "The ''master.allow-header-cert-info'' setting is deprecated.")
-                            (i18n/trs "Remove it, set ''jruby-puppet.use-legacy-auth-conf'' to ''false'', migrate your authorization rule definitions in the /etc/puppetlabs/puppet/auth.conf file to the /etc/puppetlabs/puppetserver/conf.d/auth.conf file, and set ''authorization.allow-header-cert-info'' to the desired value.")))
-          (log/warn (format "%s %s"
-                            (i18n/trs "The ''master.allow-header-cert-info'' setting is deprecated and will be ignored in favor of the ''authorization.allow-header-cert-info'' setting because the ''jruby-puppet.use-legacy-auth-conf'' setting is ''false''.")
-                            (i18n/trs "Remove the ''master.allow-header-cert-info'' setting.")))))
+        (log/warn (format "%s %s"
+                          (i18n/trs "The ''master.allow-header-cert-info'' setting is deprecated and will be ignored in favor of the ''authorization.allow-header-cert-info'' setting .")
```

Comment:
Oooo, is this something else we can rip out? (As part of another pr)

Reply:
I was hoping so.